.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["@grafana/eslint-config"],
+  "root": true,
+  "plugins": ["no-only-tests", "@emotion", "lodash"],
+  "rules": {
+    "no-only-tests/no-only-tests": "error",
+    "react/prop-types": "off",
+    "@emotion/jsx-import": "error",
+    "lodash/import-scope": [2, "member"]
```

Comment:
just curious what inspired the need to extend what we have? Seems like we'd want to keep things more or less in line with whatever the defaults right?

Reply:
I just copied this from the root of grafana/grafana but I guess it's not really that needed. I can just use the default.